global.json

```diff
@@ -1,9 +1,9 @@
 {
   "tools": {
-    "dotnet": "3.1.112",
+    "dotnet": "3.1.113",
     "runtimes": {
       "dotnet": [
-        "2.1.25",
+        "2.1.26",
```

Comment:
Why doesn't https://dotnet.microsoft.com/download/dotnet/2.1 show this runtime version yet❔

Reply:
It does now